<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-UPI6MMU" failures="2" tests="6" name="Surefire test" time="24.127" errors="0" timestamp="08 Jun 2025 14:53:39 GMT">
  <testcase classname="swaglab_test.LoginTest" name="LoginSuccessTest" time="0.524">
    <failure type="java.lang.AssertionError" message="expected [Product] but found [Products]">
      <![CDATA[java.lang.AssertionError: expected [Product] but found [Products]
at swaglab_test.LoginTest.LoginSuccessTest(LoginTest.java:30)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- LoginSuccessTest -->
  <testcase classname="swaglab_test.LoginTest" name="LockedUserLoginFailureTest" time="0.493">
    <failure type="java.lang.AssertionError" message="expected [Epic sadface: Sorry, this user has been locked out.] but found [Epic sadface: Username and password do not match any user in this service]">
      <![CDATA[java.lang.AssertionError: expected [Epic sadface: Sorry, this user has been locked out.] but found [Epic sadface: Username and password do not match any user in this service]
at swaglab_test.LoginTest.LockedUserLoginFailureTest(LoginTest.java:58)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- LockedUserLoginFailureTest -->
  <testcase classname="swaglab_test.LoginTest" name="LoginFailureTestFromExcel" time="0.471"/>
  <testcase classname="swaglab_test.LoginTest" name="LoginFailureTestFromProperties" time="0.456"/>
  <testcase classname="swaglab_test.ProductTest" name="addToCartTest" time="1.108"/>
  <testcase classname="swaglab_test.LoginTest" name="LoginFailureTest" time="0.584"/>
  <testcase classname="swaglab_test.BaseClass" name="@AfterTest CloseExcel" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest CloseExcel -->
  <testcase classname="swaglab_test.BaseClass" name="@BeforeTest SetUpExcel" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest SetUpExcel -->
</testsuite> <!-- Surefire test -->
